changelog.md

```diff
@@ -301,6 +301,7 @@
 
 - Added `unsafeIsolate` and `extract` to `std/isolation`.
 
+- `--hint:CC` now goes to stderr (like all other hints) instead of stdout .
```

Comment:
```suggestion
- `--hint:CC` now goes to stderr (like all other hints) instead of stdout.
```

Oops - I introduced this in my previous suggestion. Sorry.

Reply:
can be fixed in subsequent PR

Reply:
ugh, CI failed with unrelated `##[error]The operation was canceled.` (but possibly related to https://github.com/nim-lang/Nim/issues/17407), so i'm merging your change to restart CI